plugins/ua_securitypolicy_basic128rsa15.c

```diff
@@ -13,11 +13,13 @@
 #include <mbedtls/entropy.h>
 #include <mbedtls/entropy_poll.h>
 #include <mbedtls/error.h>
-#include <ua_plugin_pki.h>
 
+#include "ua_plugin_pki.h"
+#include "ua_plugin_securitypolicy.h"
 #include "ua_securitypolicy_basic128rsa15.h"
 #include "ua_types.h"
 #include "ua_types_generated_handling.h"
+#include "../src/ua_securechannel.h"
```

Comment:
This is an internal API. You can't use it from here.
What do you need that is part of the SecureChannel?

Reply:
I refactored the getEncryptionOverhead function out of the securitypolicy into the securechannel since it is the same for all policies. We could make it part of a ua_plugin_securitypolicy.c file i guess?